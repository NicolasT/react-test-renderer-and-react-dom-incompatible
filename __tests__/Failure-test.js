import renderer from 'react/lib/ReactTestRenderer';

/* Imagine the following import is actually burried somewhere in the dependency
 * chain of some module I actually want to test...
 */
import ReactDOM from 'react-dom';

describe('Failure', () => {
  it('fails', () => {
  });
});
