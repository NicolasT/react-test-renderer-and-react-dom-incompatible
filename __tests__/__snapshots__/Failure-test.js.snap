exports[`<HelloWorld /> renders correctly 1`] = `
<div>
  Hello, world!
</div>
`;
